streams/src/test/java/org/apache/kafka/test/MockProcessorContext.java

```diff
@@ -305,4 +328,12 @@ public ProcessorNode currentNode() {
     public void close() {
         metrics.close();
     }
+
+    private static final class NoOpRestoreListener extends AbstractNotifyingBatchingRestoreCallback {
```

Comment:
`org.apache.kafka.streams.processor.internals.NoOpStateRestoreListener` can be used here?